react/features/base/dialog/components/native/InputDialog.js

```diff
@@ -118,6 +118,16 @@ class InputDialog extends BaseDialog<Props, State> {
      * @returns {void}
      */
     _onChangeText(fieldValue) {
+        const inputKeyboardType = this.props.textInputProps
```

Comment:
I don't think this change belongs here. `InputDialog` is a dialog for generic text input. `,` and `.` may have relevance in some other context. I'd say it's the responsibility of `RoomLockPrompt`, since it's the one which interprets the input as a password.

Reply:
All comments are fixed except this one. Not sure how to do that change other than duplicating a lot of code.

Reply:
Butting in with my stinky opinion: a changeset with a small footprint would be InputDialog taking in a validation/filtering function. so it'd be like the HTML Input pattern attribute. The function could be run after the change event but before state is set. If you feel like I could see you taking the changes further, like allowing InputDialog to have a controlled input.

Reply:
ok, I can try that.

Reply:
@saghul I did the change, do approve now?